package.json

```diff
@@ -45,7 +45,7 @@
     "prettier": "^2.8.4",
     "pretty-quick": "^3.1.3",
     "turbo": "^1.8.5",
-    "typescript": "4.6.4",
+    "typescript": "4.7.4",
```

Comment:
guess we can bring this up to 5.5

Reply:
As per our conversation this will be done in future PBI